docs/logging.rst

```diff
@@ -12,20 +12,27 @@ This example is taken from the `sample ERC20 contract <https://github.com/vyperl
 ::
 
     # Events of the token.
-    Transfer: event({_from: indexed(address), _to: indexed(address), _value: uint256})
-    Approval: event({_owner: indexed(address), _spender: indexed(address), _value: uint256})
+    event Transfer:
+        sender: indexed(address)
+        receiver: indexed(address)
+        value: uint256
+
+    event Approval:
+        owner: indexed(address)
+        spender: indexed(address)
+        value: uint256
 
     # Transfer some tokens from message sender to another address
     def transfer(_to : address, _value : uint256) -> bool:
 
        ... Logic here to do the real work ...
 
        # All done, log the event for listeners
-       log.Transfer(msg.sender, _to, _amount)
+       log Transfer(msg.sender, _to, _value)
 
 Let's look at what this is doing. First, we declare two event types to log. The two events are similar in that they contain
 two indexed address fields. Indexed fields do not make up part of the event data itself, but can be searched by clients that
-want to catch the event. Also, each event contains one single data field, in each case called ``_value``. Events can contain several arguments with any names desired.
+want to catch the event. Also, each event contains one single data field, in each case called ``value``. Events can contain several arguments with any names desired.
```

Comment:
Name doesn't match above

Reply:
The name in the event is `value`, the local memory variable is `_value`.